### DISCUSSION TITLE
Cluster sub menu aligned to the right

### DISCUSSION THREAD

**Discussion by markdynamo6** [filamentphp/filament#14043] [Help] [answered]
### Package

Panel builder

### Package Version

v3

### How can we help you?

Is there a way to have the cluster sub menu _after_ the main content, rather than next to the main nav as it is now?

**Comment by e2d2-dev** [comment, marked as answer]
Hi,

Adding

`    protected static SubNavigationPosition $subNavigationPosition = SubNavigationPosition::End;
`

should do the job.


ciao ed

**Reply by markdynamo6** [reply]
Awesome. Thanks Ed
